### ISSUE TITLE
segmantion train error "isinstance(x, (list,tuple)) and len(x)<=2, 'incorrect input"

### ISSUE THREAD

**Issue by skoomn** [TexasInstruments/jacinto-ai-devkit#10]
when i traing segmentation i meet a problem: 
when i set batch_size=8 and iter_size=1,program case a problem "isinstance(x, (list,tuple)) and len(x)<=2, 'incorrect input'".
"isinstance(x, (list,tuple)) and len(x)<=2, 'incorrect input" is found in the file "/modules/pytorch_jacinto_ai/xvision/models/pixel2pixel/deeplabv3lite.py",local in line 106.
when i set batch_size=1 and iter_size=1 the model can be trained. Is the segmantation model must be trained one image by one?
can you tell me how can set batch size bigger than one or why i must set batch size to be one?
As i konow the model with "batchnorm" need a better result,usually need input images number bigger than one.
(other set  if need ,model name is deeplabv3lite_mobilenetv2_tv i only have one gpu card)
